### PR TITLE
Adds hydroponics tools and shovels to autolathe_designs.dm

### PR DESCRIPTION
**Pull request by erwgd**
Cultivators, spades, shovels, plant analyzers and hatchets can now be made in the autolathe.

:cl: Erwgd
rscadd: The autolathe can now make hydroponics tools! Access the design routines in the Misc. category of the machine.
/:cl:

### CODE REVIEW

**Comment by Aloraydrel** [comment]
Doing gods works

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -215,6 +215,38 @@
...
+	materials = list(MAT_METAL = 30, MAT_GLASS = 20)
+	build_path = /obj/item/device/analyzer/plant_analyzer
+	category = list("initial","Misc")
+
+/datum/design/shovel
+	name = "Shovel"
+	id = "shovel"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 50)
+	build_path = /obj/item/weapon/shovel/
```
Comment (Fox-McCloud) [inline code/modules/research/designs/autolathe_designs.dm]: extra `/` on the end here.

**Comment by Fox-McCloud** [comment]
Don't forget hatchets.

File: code/modules/research/designs/autolathe_designs.dm [hunk 2]
```diff
@@ -215,6 +215,46 @@
...
+
+/datum/design/spade
+	name = "Spade"
+	id = "spade"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 50)
+	build_path = /obj/item/weapon/shovel/spade
+	category = list("initial","Misc")
+
+/datum/design/hatchet
```
Comment (octareenroon91) [inline code/modules/research/designs/autolathe_designs.dm]: Hatchets shouldn't be too plentiful. Maybe also push the other designs to use about 200 material.

**Pull request merged**